tests/backup/backup_test.go

```diff
@@ -2052,7 +2052,7 @@ var _ = Describe("{ShareBackupAndEdit}", func() {
 	providers := getProviders()
 	JustBeforeEach(func() {
 		StartTorpedoTest("ShareBackupAndEdit",
-			"Share backup with restore and full access mode and edit the shared backup", nil, 0)
+			"Share backup with restore and full access mode and edit the shared backup", nil, 82950)
```

Comment:
In this test you are trying to create cloud credentials with the same name. Since there is no delay between the creation of  `credName` and `cred1Name` it is getting created with the same name and failing. Please see the log and fix it.

Reply:
Creating cred1Name after creating backuplocation